src/com/scottyplunkett/server/com/scottyplunkett/server/Router.java

```diff
@@ -0,0 +1,20 @@
+package com.scottyplunkett.server;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Router {
+        public static Path route(String requestLine) {
+            if ("GET / HTTP/1.1".equals(requestLine))
```

Comment:
If you remember our talk about conditionals - _this_ would be a great example of a conditional which could be refactored into sort of a strategy matcher / strategy handler pattern.

Reply:
Thanks for giving me direction on this, I wasn't quite sure what I wanted to do here but hate its ugliness